ngraph_creator/operations/include/OperationsBase.hpp

```diff
@@ -118,9 +118,19 @@ class OperationsBase {
     std::shared_ptr<ngraph::Node> DequantizeNode(std::shared_ptr<ngraph::Node> input, size_t index,
                                                  ngraph::element::Type dequantizeType);
 
+    const Operand& getInputOperand(uint32_t index) {
```

Comment:
Do we need this , as there are helper APIs to get the type, dims etc for the operands.

Reply:
Function is added to provide access to operand data type especially if the operand is referenced multiple times in the given function.